src/Client/ConnectionParameters.cpp

```diff
@@ -103,6 +103,9 @@ ConnectionParameters::ConnectionParameters(const Poco::Util::AbstractConfigurati
 #endif
     }
 
+    proto_send_chunked = config.getString("proto_caps.send", "notchunked_optional");
+    proto_recv_chunked = config.getString("proto_caps.recv", "notchunked_optional");
```

Comment:
So, we want to be able to set these protos using config. Do we need to document it somewhere or it will be used only internally? And what `caps` means?